src/getting-started.md

```diff
@@ -131,204 +125,189 @@ are two major downsides:
 
     Q: Bonjour
     A: Hello
        Page #12
 
-To do this, we can set the question and answer templates to:
+To do this, we can set the answer template to:
 
     Q: {{French}}
     A: {{English}}<br>
        Page #{{Page}}
 
-By surrounding a field name in double curly brackets, we tell Anki to
-replace that section with the actual information in the field. Anything
-not surrounded by curly brackets remains the same on each card. (For
-instance, we don’t have to type “Page \#” into the Page field when
-adding material – it’s added automatically to every card.) &lt;br&gt; is
-a special code that tells Anki to move to the next line; more details
-are available in the [templates](templates/intro.md) section.
+In card templates, field names are wrapped in double curly brackets like `{{French}}` or `{{English}}`. Anki replaces those with the actual text the field contain. This is known as ["Field replacement"](/field-replacement.md). Text
+not surrounded by curly brackets appears the same on each card. For
+example, we don't need to enter "Page \#" in every card; it’s added automatically to every card. The `&lt;br&gt;` tag is
+a special code that tells Anki to move to the next line. For details, see the [templates](templates/intro.md) section.
 
-The production card templates work in a similar way:
+The production card's templates will also work in a similar way:
 
     Q: {{English}}
     A: {{French}}<br>
        Page #{{Page}}
 
-Once a card type has been created, every time you add a new note, a card
+After a card type has been created, every time you add a new note, a card
 will be created based on that card type. Card types make it easy to keep
 the formatting of your cards consistent and can greatly reduce the
 amount of effort involved in adding information. They also mean Anki can
-ensure related cards don’t appear too close to each other, and they
+ensure related cards don't appear too close to each other, and they
 allow you to fix a typing mistake or factual error once and have all the
 related cards updated at once.
 
-To add and edit card types, click the “Cards…​” button while adding or
+To add and edit card types, click the "Cards…​" button while adding or
 editing notes. For more information on card types, please see the [Cards and Templates](templates/intro.md) section.
 
 ### Note Types
 
 Anki allows you to create different types of notes for different
 material. Each type of note has its own set of fields and card types.
-It’s a good idea to create a separate note type for each broad topic
-you’re studying. In the above French example, we might create a note
-type called “French” for that. If we wanted to learn capital cities, we
+It's a good idea to create a separate note type for each broad topic
+you’re studying. In the previous French example, we might create a note
+type called "French" for that. If we wanted to learn capital cities, we
 could create a separate note type for that as well, with fields such as
```

Comment:
I think it's unnecessary, and to my ears "world capital cities" sounds weird (probably because here we say country capital/state capital).

Let's also keep the "as well"; it flows well if you read the whole thing.

Reply:
"capital cities of the world"?

I think we don't need both "separate" and "as well" -- both of them are doing the same job.

Reply:
I don't see the point in perfectionism.

I don't know, "seperate" and "as well" are different things for me.

Reply:
```suggestion
type called "French" for that. If we wanted to learn capital cities, we
could create a note type for that as well, with fields such as
```